### PR TITLE
Declare the event type bit size

### PR DESCRIPTION
**Pull request by huntc**
Declares the number of bits used for the event type.

Requires: https://github.com/akka/akka-edge-rs/pull/144

### CODE REVIEW

File: samples/grpc/iot-service-rs/Cargo.toml
Comment (huntc) [inline samples/grpc/iot-service-rs/Cargo.toml:1]: The change here is temporary and will need to reflect an 0.8.0 version when released.

Comment (patriknw) [reply samples/grpc/iot-service-rs/Cargo.toml:1]: 0.8.0 has been released

Comment (huntc) [reply samples/grpc/iot-service-rs/Cargo.toml:1]: @patriknw Updated and ready for review/merge.

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**